js/ui/color_box/color_box.js

```diff
@@ -319,6 +319,9 @@ const ColorBox = DropDownEditor.inherit({
 
         this.option('text', text);
 
+        const $input = this._input();
+        this.setAria('live', 'polite', $input);
```

Comment:
NVDA reader doesnt't read the value on meter key when color box popup is closed. But i'n not sure this is a problem